CSharp/DevVmPowershell/DevVmPsModules/Cmdlets/AddApplicationFromLibraryModule.cs

```diff
@@ -75,10 +75,15 @@ protected override void ProcessRecordCode()
 				relativityAdminPassword: RelativityAdminPassword,
 				sqlAdminUserName: SqlAdminUserName,
 				sqlAdminPassword: SqlAdminPassword);
-			IApplicationInstallHelper applicationInstallHelper = new ApplicationInstallHelper(connectionHelper);
+			ISqlRunner sqlRunner = new SqlRunner(connectionHelper);
+			ISqlHelper sqlHelper = new SqlHelper(sqlRunner);
+			RestHelper restHelper = new RestHelper();

```

Comment:
- [ ] Aaron created an interface IRestHelper, Can we use that instead to pass to the various Helper constructors?